src/apps/stable/routes/_redirects.ts

```diff
@@ -0,0 +1,6 @@
+import type { Redirect } from 'components/router/Redirect';
+
+export const REDIRECTS: Redirect[] = [
+    { from: 'mypreferencesquickconnect.html', to: '/quickconnect' },
```

Comment:
`{ from: 'quickconnect', to: '/quickconnect' }`

Reply:
The "from" value is the old url of the page. This way if someone tries to go to the old url directly they get bounced to the new url instead of landing on a broken page.